addons/wowl/static/src/components/navbar/navbar.ts

```diff
@@ -8,12 +8,20 @@ export class NavBar extends Component<{}, OdooEnv> {
   menuRepo = useService("menus");
   state = useState({ showDropdownMenu: false });
 
-  systrayItems = this.env.registries.systray.getAll();
+  systrayItems = this._getSystrayItems();
 
   toggleDropdownMenu() {
     this.state.showDropdownMenu = !this.state.showDropdownMenu;
   }
 
+  _getSystrayItems() {
+    return this.env.registries.systray.getAll().sort((x, y) => {
+      x.sequence = x.sequence === undefined ? 50 : x.sequence;
```

Comment:
you modify the content of the registry. Usually, not a good idea. You could simply here do this:
```ts
  const xSeq = ...;
  const ySeq = ...;
  return ySeq - xSeq;
```